### DISCUSSION TITLE
I pay a suscription with my credit card but the wrong email HELP!

### DISCUSSION THREAD

**Discussion by Iamthespaceship** [aclap-dev/video-downloadhelper#1067] [Help and Q&A]
### Your question

The email that i wite on my suscription it's not mine, I don have access on that email, I made i mistake, help please

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: es-ES
coapp: "checking"
license: {"checking":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "C:\\Users\\JAVIER\\Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 22,
    "successfull_dl": 41
}

**Comment by jcfrog** [comment]
Hi,
Please send us an email at licenses@downloadhelper.net with the address you used and the one you want.

**Reply by Iamthespaceship** [reply]
Done I send it
